handle/handle.go

```diff
@@ -0,0 +1,21 @@
+//Pachage handle has handle, which convert
+//input in necessary values.
+package handle
+
+import (
+	"flag"
+	"github.com/shopspring/decimal"
+	"volume_calculator/calculator"
+)
+
+func Handle() *calculator.Calculator{
+	var shape string
+	flag.StringVar(&shape, "shape", "string", "a string")
+	radius, _ := decimal.NewFromString(*flag.String("radius", "", "a float64"))
+	length, _ := decimal.NewFromString(*flag.String("length", "", "a float64"))
+	width, _ := decimal.NewFromString(*flag.String("width", "", "a float64"))
+	hieght, _ := decimal.NewFromString(*flag.String("hieght", "", "a float64"))
+	flag.Parse()
```

Comment:
This code basically relates only to cli. Probably in the future we will have build in http server as a source of these params so, the logic of `interface` layer should be decoupled from business logic.

1. I'd propose create an internal struct which will hold length, width, hieght ... etc on the business logic level.
2. Create a new package `cli` which will incapsulate all logic related to command line things. That package may have a `Load()` method which does all work with flags and returns a struct from step 1. 
3. Pass that struct to the `Builder` which will create a necessary `shape`
4. Call `calculateValue()` and print the result